plugins/woocommerce-admin/client/profile-wizard/steps/industry.js

```diff
@@ -45,8 +45,9 @@ const Loader = ( props ) => {
 class Industry extends Component {
 	constructor( props ) {
 		const profileItems = get( props, 'profileItems', {} );
-		let selected = profileItems.industry || [];
-
+		let selected = Array.isArray( profileItems.industry )
+			? [ ...profileItems.industry ]
```

Comment:
I'm curious, why do we need to clone the value here? Is there an issue to reuse the same object reference?

Reply:
Yes, it's the same object reference issue.

Previously, we used the same object in the local state; when we update the local "selected" state, it persists the changes in the global profileItems. So when a user discards the changes and navigates back from a different screen, the user would still see the "changed" view.

Reply:
Brilliant, thanks!